UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Filters/SecurityRequirementsOperationFilter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
```

Comment:
These aren't used and could be removed.

Reply:
Do you mean namespace System.Collections.Generic or something else? This namespace is used for list operation.Security = new List<OpenApiSecurityRequirement>

Reply:
Visual Studio was highlighting the using's as not being needed..

Reply:
Resolving this conversation as this namespace is required for List